token-allocation/README.md

```diff
@@ -22,7 +22,7 @@ The adapter takes the following environment variables:
 ## Input Params
 
 - `components`: Array of the token symbols. 
-- `units` (optional): Array of balances in wei of each token. `10000000000000000000` wei (1 ether) for each token by default.
+- `units` (optional): Array of balances of each token in ETH. 1 ether for each token by default.
```

Comment:
Maybe we should just make this as generic as possible? No mention of ETH, just whole units.

Reply:
Yeah, no need to mention ETH here. These are just whole-number units that we convert to currency using 18 decimals by default if not specified differently.